core/tests/protractor_utils/users.js

```diff
@@ -46,16 +48,18 @@ var login = async function(
       }
       return true;
     }, waitFor.DEFAULT_WAIT_TIME_MSECS, 'Login takes too long.');
-
-  await (await driver.findElement(protractor.By.name('email'))).clear();
-  await (await driver.findElement(protractor.By.name('email'))).sendKeys(email);
+  var emailInput = element(protractor.By.name('email'));
+  await action.clear('Email input', emailInput);
+  await action.sendKeys('Email input', emailInput, email);
   if (isSuperAdmin) {
-    await (await driver.findElement(protractor.By.name('admin'))).click();
-    let adminCheckboxStatus = await driver.findElement(
-      protractor.By.name('admin')).getAttribute('checked');
+    var testAdminButton = element(protractor.By.name('admin'));
```

Comment:
```suggestion
    var adminCheckbox = element(protractor.By.name('admin'));
```

This is a checkbox and not a button.

Reply:
Fixed.